barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover.hpp

```diff
@@ -79,14 +80,6 @@ template <class ProverInstances_> class ProtoGalaxyProver_ {
      */
     void prepare_for_folding();
 
-    /**
-     * @brief Send the public data of an accumulator, i.e. a relaxed instance, to the verifier (ϕ in the paper).
-     *
-     *  @param domain_separator separates the same type of data coming from difference instances by instance
-     * index
-     */
-    void send_accumulator(std::shared_ptr<Instance>, const std::string& domain_separator);
```

Comment:
Accumulator is the output of previous folding iteration so need for the prover to send it in the transcript after the current changes